src/treemodel/delta/inserttextdelta.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+CKEDITOR.define( [
+	'treemodel/delta/delta',
+	'treemodel/delta/register',
+	'treemodel/operation/insertoperation',
+	'treemodel/nodelist'
+], ( Delta, register, InsertOperation, NodeList ) => {
+	/**
+	 * To provide specific OT behavior and better collisions solving, the {@link treeModel.Batch#insert} method
+	 * uses the `InsertTextDelta` class which inherits from the `Delta` class and may overwrite some methods.
+	 *
+	 * @class treeModel.delta.InsertTextDelta
+	 */
+	class InsertTextDelta extends Delta {}
+
+	/**
+	 * Inserts a node or nodes at the given position. The nodes will have same attributes as the current attributes of
+	 * {@link treeModel.Document#selection document selection}. Commonly used for typing or plain-text paste (without formatting).
+	 *
+	 * @chainable
+	 * @memberOf treeModel.Batch
+	 * @method insertText
+	 * @param {treeModel.Position} position Position of insertion.
+	 * @param {treeModel.Node|treeModel.Text|treeModel.NodeList|String|Iterable} nodes The list of nodes to be inserted.
+	 * List of nodes can be of any type accepted by the {@link treeModel.NodeList} constructor.
+	 */
+	register( 'insertText', function( position, nodes ) {
```

Comment:
Honestly, I don't know. All names seems really underwhelming. `insertWithoutAttrs` is closest to what the delta really does and `insertPlain` does not mean anything.

There is one more thing, though. Wasn't the difference between this delta and `InsertDelta` in that elements inserted by `InsertTextDelta` are moved when they are in moved range? Normal behavior for inserting is that if you insert elements into range that got moved, they are inserted at the move `sourcePosition` (moving [] to another `<p>`):
`<p>fo[o^ba]r</p><p></p>` => `<p>foxyzr</p><p>oba</p>`

This delta was supposed to move inserted elements if range got moved:
`<p>fo[o^ba]r</p><p></p>` => `<p>for</p><p>oxyzba</p>`? Wasn't that more natural for typing?

If that's the case maybe we should acknowledge also this behavior. Maybe `insertWeak`  would be most appropriate name?